README.md

```diff
@@ -38,7 +38,7 @@
 
 ### [Antigen](https://github.com/zsh-users/antigen)
 
-Add `antigen bundle veelenga/crystal.zsh` to your `.zshrc` with your other bundle commands. Antigen will handle cloning the plugin for you automatically the next time you start zsh. You can also add the plugin to a running zsh with `antigen bundle veelenga/crystal.zsh` for testing before adding it to your `.zshrc`.
+Add `antigen bundle veelenga/crystal-zsh` to your `.zshrc` with your other bundle commands. Antigen will handle cloning the plugin for you automatically the next time you start zsh. You can also add the plugin to a running zsh with `antigen bundle veelenga/crystal.zsh` for testing before adding it to your `.zshrc`.
```

Comment:
`antigen bundle veelenga/crystal.zsh` appears twice on this line, please correct both.